ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/checkbox/v1/checkbox/_cq_dialog/.content.xml

```diff
@@ -76,20 +76,20 @@
                                                   fieldLabel="When Checked, return value"
                                                   fieldDescription="Specify the value to be returned when the checkbox is checked by the user."
                                                   name="./checkedValue"/>
-                                                <enableUncheckedValue
+                                                <preserveUncheckedStateValue
```

Comment:
@rismehta This is updated in checkbox component also in this PR because switch component inherits dialog from checkbox